src/VisualStudio/Core/Impl/CodeModel/InternalElements/AbstractCodeType.cs

```diff
@@ -123,7 +130,7 @@ public EnvDTE.CodeNamespace Namespace
         {
             get
             {
-                var namespaceNode = GetNamespaceOrTypeNode();
```

Comment:
Should probably use links on http://source.roslyn.io here, but oh, of course.  I even went and looked there and misinterpreted the results.  :blush: